src/frontend/Player/createPlayer.ts

```diff
@@ -16,9 +37,9 @@ export const createPlayer: VideoPlayerCreator = (
         video.urls!,
         video.live_state,
       );
-      return {
+      return Promise.resolve({
         destroy: () => player.dispose(),
-      };
+      });
```

Comment:
I don't think you have to wrap this return statement in a `Promise.resolve`.

Reply:
Just a `Promise` object ? I need to return a promise here

Reply:
Your function is async so `createPlayer`  returns a promise implicitly.

Reply:
Indeed you are right !

Reply:
done